content/reference/endpoints/rentals.md

```diff
@@ -170,6 +170,43 @@ GET /rentals
 
 <%= render 'json_response', endpoint: "rentals", scopes: %w(public rentals_read-rentals_write) %>
 
+## Search rentals
+
+Search parameters allow to filter rentals by specified fields.
+
+Example:
+
+~~~
+GET /rentals/search?start_at=20140324&end_at=20140330
+~~~
+
+### Search Parameters
+
+Name              | Type    | Default | Description
+------------------|---------|--------------
+amenities         | String  |         | Rentals with given amenities, each amenity has to be a separate parameter.
```

Comment:
Indeed, just hoped we could with this before https://github.com/BookingSync/BookingSync/issues/608